CONTRIBUTING.md

```diff
@@ -152,35 +152,35 @@ The SDK uses the following pattern for namespace name: **olp**::*component::subc
 
 #### API Class Name
 
-The SDK uses the following pattern for API class name : *Api***Client**, e.g. `StreamLayerClient`
+The SDK uses the following pattern for API class name: *Api***Client**, for example, `StreamLayerClient`.
 
 #### API Method Signature
 
-The SDK provides a set of common classes that are used when defining the APIs.
+The SDK provides a set of standard classes that are used to define the APIs.
```

Comment:
`common` is a more used term for such classes, or `generic`
`standard` might be mistaken with the std library classes which is something totally different.